app/api/student_api/sygne.rb

```diff
@@ -3,7 +3,7 @@
 module StudentApi
   class Sygne < Base
     def endpoint
-      base_url + format("etablissements/%s/eleves/", @establishment.uai)
+      base_url + format("etablissements/%s/eleves/", uai)
```

Comment:
Je mettrais `@uai` ici pour bien signigier que c'est une variable d'instance

Reply:
effectivement c'est mon code initial mais il vaut mieux passer par l'accessor de `attr_reader` (qu'on pourrait avoir besoin d'override un jour) : ça isole mieux la donnée "pure".

c.f la section "Depend on Behavior, Not Data" de POODR